src/lib.rs

```diff
@@ -977,6 +977,21 @@ pub fn version_is_supported(version: u32) -> bool {
     }
 }
 
+macro_rules! add_frame {
+    {$frames:expr, $frame:expr, $payload_len: expr, $left:expr} => {{
```

Comment:
Why are you using `{ }` instead of `( )`? `()` is more consistent with other macros and you can call this more like a function: `add_frame!(...)`.

Reply:
Done. Somehow I had errors when I use `(` but it seems ok actually...